src/status_im/ui/screens/communities/discover_communities.cljs

```diff
@@ -9,23 +9,22 @@
             [quo2.components.buttons.button :as quo2.button]
             [quo2.components.counter.counter :as quo2.counter]
             [quo2.components.tags.tags :as tags]
-            [quo2.foundations.colors :as quo2.colors]
             [quo.components.safe-area :as safe-area]
             [quo2.components.tabs.tabs :as quo2.tabs]
-            [status-im.ui.screens.chat.photos :as photos]
             [status-im.react-native.resources :as resources]
-            [status-im.multiaccounts.core :as multiaccounts]
             [status-im.ui.components.topbar :as topbar]
-            [status-im.ui.components.plus-button :as plus-button]
             [status-im.utils.handlers :refer [<sub  >evt]]
             [status-im.ui.components.topnav :as topnav]
```

Comment:
should this not be using new [status-im.ui.components.topnav :as topnav] instead?